docs/getting_started/tre_organisation.md

```diff
@@ -18,17 +17,37 @@ You can use your home folder for any files you want to keep, but it is not backe
 Folders are suffixed with `red` or `green` to indicate the type of data that is stored there. Red is for potentially sensitive data that should not be shared without the outside world. Green is for data that can be shared with the outside world.
 
 ## library-red 
-Available at `/library-red` in your sandbox, this is a read-only folder that is shared between all users. This contains the data that you will want to use. 
+Available at `/genesandhealth/library-red` in your sandbox, this is a read-only folder that is shared between all users. This contains the data that you will want to use.
 
-`library-red` is slower storage of large capacity (\>8 PiB @ Feb 2022). For large files, the whole file needs to be read and cached first by gcsfuse, fileseek to a certain part of the file is not possible. For high performance/large files it may be better to make a copy to /red or /home/ivm.
+`library-red` is slower storage of large capacity (\>8 PiB @ Feb 2022). For large files, the whole file needs to be read and cached first by gcsfuse, fileseek to a certain part of the file is not possible. For high performance/large files it may be better to make a copy to `/genesandhealth/red` or `/home/ivm`.
 
 `library-red` is a google storage bucket gs://qmul-sandbox-production-library-red/ (read+write only for admins)
 
 `library-red` is where curated and raw data is stored. This is where you will find the data you need to run your analysis. There are a number of subfolders in `library-red` that contain different types of data, and each folder should be used for a specific purpose. If you find a folder that does not have a readme file, please contact the Genes and Health team to get more information on what the folder is for.
 
 ## red
-`red` is used directly by the virtual machine, and is specific to each sandbox. Users in the same sandbox can all see the contents of the red folder. Most organisations use this folder to store and run their analysis. 
+`red` is used directly by the virtual machine, and is specific to each sandbox. Users in the same sandbox can all see the contents of the red folder. Most organisations use this folder to store their analyses.
+
+`red` has slower storage than `/home/ivm` but is backed up, and we strongly recommend that you use this folder to store your data and any analysis files you are running frequently to avoid losing your work.
```

Comment:
Make it clear that data that won't be used during some job execution should go in red, and anything they're actively working on should go in /home/ivm

Reply:
added.